### DISCUSSION TITLE
Fix for #3935 not in 4.0.2?

### DISCUSSION THREAD

**Discussion by maxstolze** [eclipse-rdf4j/rdf4j#3971] [Q&A] [answered]
Was just wondering, why the fix for #3935 didn't make it into the 4.0.2. Is there a reason? It still keeps us blocked from using 4.0 in our project. Thanks in advance!

**Comment by hmottestad** [comment, marked as answer]
Looks like it's a feature/enhancement and not a bug so it's targeted for 4.1.0.

**Reply by maxstolze** [reply]
For us it is a bug, as without the fix it always runs into an `UnexpectedRollBackException`. It seemed to be labeled wrong by me in the first place. (False Issue Template)
